packages/next-ui/TextInputNumber/TextInputNumber.tsx

```diff
@@ -141,6 +141,7 @@ export function TextInputNumber(props: TextInputNumberProps) {
       }}
       inputProps={{
         ...inputProps,
+        'aria-label': i18n._(/* i18n */ 'Quantity'),
```

Comment:
TextInputNumber of course doesn't always have to be Quantity?

Reply:
Changed to number